### PR TITLE
Update User-Forum link

### PR DESCRIPTION
**Pull request by AI-Pranto**
1 update user forum link 
  - index.rst
  - troubleshoot.rst

2 update install.rst. I didn't notice it in my first commit.

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -15,8 +15,7 @@ OpenMC was originally developed by members of the `Computational Reactor Physics
 Group <http://crpg.mit.edu>`_ at the `Massachusetts Institute of Technology
 <http://web.mit.edu>`_ starting in 2011. Various universities, laboratories, and
 other organizations now contribute to the development of OpenMC. For more
-information on OpenMC, feel free to send a message to the User's Group `mailing
-list <https://groups.google.com/forum/?fromgroups=#!forum/openmc-users>`_.
+information on OpenMC, feel free to send a message to the User's Group `OpenMC Discourse Forum <https://openmc.discourse.group/>`_.
```
Comment (paulromano) [inline docs/source/index.rst:18]:
```suggestion
information on OpenMC, feel free to post a message on the `OpenMC Discourse
Forum <https://openmc.discourse.group/>`_.
```

File: docs/source/usersguide/troubleshoot.rst [hunk 2]
```diff
@@ -31,7 +31,7 @@ on. Create a new build directory and type the following commands:
 
 Now when you re-run your problem, it should report exactly where the program
 failed. If after reading the debug output, you are still unsure why the program
-failed, send an email to the OpenMC User's Group `mailing list`_.
+failed, send an email to the OpenMC User's Group `OpenMC Discourse Forum <https://openmc.discourse.group/>`_.
```
Comment (paulromano) [inline docs/source/usersguide/troubleshoot.rst:34]:
```suggestion
failed, post a message on the `OpenMC Discourse Forum
<https://openmc.discourse.group/>`_.
```

File: docs/source/usersguide/install.rst [hunk 3]
```diff
@@ -40,6 +40,26 @@ It is possible to list all of the versions of OpenMC available on your platform
...
+
+    conda create -n openmc-env openmc
+    
+This will install OpenMC in a conda environment called `openmc-env`. To activate
+the environment, run:
+
+.. code-block:: sh
+
+    conda activate openmc-env
+
```
Comment (paulromano) [inline docs/source/usersguide/install.rst:62]: Some of this information is redundant with what appears above. I would just change this section to match what is currently in `quickinstall.rst`.

Comment (AI-Pranto) [reply docs/source/usersguide/install.rst]: Thanks @paulromano.

**Pull request merged**